instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/BaseTracer.java

```diff
@@ -187,15 +203,25 @@ public void addThrowable(Span span, Throwable throwable) {
     span.recordException(throwable);
   }
 
-  public static <C> Context extract(C carrier, TextMapPropagator.Getter<C> getter) {
+  /** @deprecated We should eliminate all static usages so we can use the non-global propagators. */
+  @Deprecated
+  public static <C> Context extractWithGlobalPropagators(
```

Comment:
how's this approach, @trask ? There's only one usage of it in the project at the moment.

Reply:
awesome